### PR TITLE
Fixes airtight plasticflaps

### PR DESCRIPTION
**Pull request by SodiumShine**
Makes them airtight on initialization as they should be.

### CODE REVIEW

File: code/game/objects/structures/plasticflaps.dm [hunk 1]
```diff
@@ -81,3 +81,7 @@
 
 /obj/structure/plasticflaps/airtight // airtight defaults to on 
 	airtight = 1
+
+/obj/structure/plasticflaps/airtight/Initialize()
+	. = ..()
+	become_airtight()
```
Comment (PsiOmegaDelta) [inline code/game/objects/structures/plasticflaps.dm]:
```
if(airtight)
	become_airtight()
```

**Review by PsiOmegaDelta — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/plasticflaps.dm (hunk 1 above)
Comment (SodiumShine) [reply code/game/objects/structures/plasticflaps.dm]: You want that for regular plastic flaps? Only, the airtight child should always be airtight no?

Comment (PsiOmegaDelta) [reply code/game/objects/structures/plasticflaps.dm]: Ah, this should indeed instead be
```
/obj/structure/plasticflaps/Initialize()
	. = ..()
	if(airtight)
		become_airtight() 
```

Comment (SodiumShine) [reply code/game/objects/structures/plasticflaps.dm]: Actually on the subject, should the destroy() be checking for airtightness? I don't know what other things cause a turf to block air.

**Review by MistakeNot4892 — APPROVED** [review]

**Comment by comma** [comment]
why do flaps even need to touch the turf? Can't they set their own `atmos_canpass` to `CANPASS_NEVER`?

**Comment by SodiumShine** [comment]
This should be good to go now.

**Review by MistakeNot4892 — APPROVED** [review]

**Pull request merged**